<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReconOrderUI</class>
 <widget class="QDialog" name="ReconOrderUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>423</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ReconOrder</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>401</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="main_tab">
    <attribute name="title">
     <string>Snap and BG</string>
    </attribute>
    <widget class="QPushButton" name="qbutton_snap_and_correct">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>160</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>Snap and BG Correct</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="qbutton_collect_background">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>160</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>Collect Background</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="qbutton_file_browser">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>110</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLabel" name="_qlabel_bg_corr_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Background correction file</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qline_bg_corr_path">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>451</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="start_monitor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(33, 255, 6);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Monitor and reconstruct</string>
     </property>
    </widget>
    <widget class="QPushButton" name="qbutton_stop_monitor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(252, 1, 7);
border-color: rgb(252, 1, 7);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Stop Monitor</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="calibrate_lc_tab">
    <attribute name="title">
     <string>Calibrate LC</string>
    </attribute>
    <widget class="QPushButton" name="qbutton_calibrate_lc">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>160</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>Calibrate LC</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="qline_swing">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0.03</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qline_wavelength">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>532</string>
     </property>
    </widget>
    <widget class="QLabel" name="_qlabel_swing">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Swing</string>
     </property>
    </widget>
    <widget class="QLabel" name="_qlabel_wavelength">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wavelength</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_state0">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>State 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_state1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>State 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_state2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>State 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_state3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>State 3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_state4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>State 4</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_state0_lca">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_state0_lcb">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_state1_lca">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_state1_lcb">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>220</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_state2_lca">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>250</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_state2_lcb">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_state3_lca">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>280</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_state3_lcb">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>280</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_state4_lca">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>310</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_state4_lcb">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>310</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_LCA">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>LCA</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_LCB">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>LCB</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_extinction">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Extinction</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_extinction">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_state0_intensity">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>190</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_state1_intensity">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_state2_intensity">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>250</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_state3_intensity">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>280</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_state4_intensity">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>310</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_intensity">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>170</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Intensity</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="logs">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <attribute name="title">
     <string>logs</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>371</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>479</width>
        <height>369</height>
       </rect>
      </property>
      <widget class="QTextBrowser" name="log_area">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>481</width>
         <height>371</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
